src/evalfunc.c

```diff
@@ -2131,6 +2132,8 @@ static funcentry_T global_functions[] =
 			ret_getreg,	    f_getreg},
     {"getreginfo",	0, 1, FEARG_1,	    arg1_string,
 			ret_dict_any,	    f_getreginfo},
+    {"getregion",	2, 2, FEARG_2,	    arg2_string,
```

Comment:
I think this should be FEARG_1 instead of FEARG_2.  This determines which argument is used when getregion() is called using the method syntax.